packages/apps/plugins/plugin-stack/src/components/StackMain.tsx

```diff
@@ -48,10 +48,16 @@ const StackMain: FC<{ stack: StackType; separation?: boolean }> = ({ stack, sepa
   const items = stack.sections
     // TODO(wittjosiah): Should the database handle this differently?
     // TODO(wittjosiah): Render placeholders for missing objects so they can be removed from the stack?
-    .filter(({ object }) => Boolean(object));
+    .filter(({ object }) => Boolean(object))
+    .map(({ id, object }) => {
+      const rest = metadataPlugin?.provides.metadata.resolver(object.__typename ?? 'never');
```

Comment:
is this to get the placeholder for the collapsed state?

Reply:
Yes, this was an attempt to get the fallback title, but the placeholder still seemed like progress.